src/tyr/route_serializer.cc

```diff
@@ -85,7 +85,7 @@ std::string serializeDirections(Api& request) {
     case Options_Format_json:
       return valhalla_serializers::serialize(request);
     case Options_Format_pbf:
-      return request.SerializeAsString();
+      return serializePbf(request);
```

Comment:
you'll see this in a couple places. basically because pbf serialization had to get a bit more complicated i needed to throw it into a separate function